packages/astro/src/@types/astro.ts

```diff
@@ -2373,68 +2313,72 @@ export interface AstroUserConfig {
 		 * export const collections = { countries };
 		 * ```
 		 *
-		 * For more advanced loading logic, you can define an object loader. This allows incremental updates and conditional loading, and gives full access to the data store. See the API in [the RFC](https://github.com/withastro/roadmap/blob/content-layer/proposals/content-layer.md#loaders).
+		 * For more advanced loading logic, you can define an object loader. This allows incremental updates and conditional loading, and gives full access to the data store. See the API in [the draft Content Layer API RFC](https://github.com/withastro/roadmap/blob/content-layer/proposals/content-layer.md#loaders).
 		 *
-		 * ### Migrating an existing content collection to use the Content Layer API
+		 * #### Migrating a content collection to content layer
```

Comment:
I still somewhat prefer the original wording here, since it feels like abandoning content collections (the things people have in their projects) themselves, not just the underlying API powering them. Can you tell me more about what you're thinking here?

It's not a deal breaker! It does also feel a tiny bit weird to read about "content layer" as a noun without the definite article. My brain really wants to hear "the" here!

Reply:
I prefer the original too. I think this may be an older version that snuck in when I was updating from your branch.